usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -3,7 +3,7 @@
 
 from collections import OrderedDict
 from decimal import Decimal
-from django.db.models import Sum
+from django.db.models import Sum, F
```

Comment:
I know it's Django, not you, but `F`?

Reply:
The long version is in [Django Docs](https://docs.djangoproject.com/en/1.11/ref/models/expressions/#f-expressions).

The short version is that it is a workaround to do aliases with Django when you are just grabbing a column. In this use case it allows the Contract and IDV response to both use the same function to fetch the parent award details even though the query paths would be different: `award_id` for Contract vs `parent_award__award_id` for IDVs.